Sources/AccountSDKIOSWeb/Simplified Login/SimplifiedLoginManager.swift

```diff
@@ -3,6 +3,7 @@ import UIKit
 public final class SimplifiedLoginManager {
     public enum SimplifiedLoginError: Error {
         case noLoggedInSessionInSharedKeychain
+        case noClientNameProvided
```

Comment:
`noClientNameFound` maybe?

As "Provided" implies to me that a client name need to be passed in as a param. 
But I am nitpicking :D